magik-lint/src/main/java/nl/ramsolutions/sw/magik/lint/MagikLint.java

```diff
@@ -109,17 +109,20 @@ private List<MagikIssue> runCheckOnFile(
     /**
      * Show checks active and inactive checks.
      *
+     * @param writer Writer Write to write output to.
+     * @param showDisabled Boolean Boolean to show disabled checks or not.
      * @throws ReflectiveOperationException -
      * @throws IOException -
      */
-    void showChecks(final Writer writer) throws ReflectiveOperationException, IOException {
-        final Iterable<MagikCheckHolder> holders = MagikLint.getAllChecks(this.config);
+    void showChecks(final Writer writer, final Boolean showDisabled) throws ReflectiveOperationException, IOException {
```

Comment:
Why the uppercase/class `Boolean`, and not the lowercase/primitive `boolean`?

Reply:
No specific reason for that, I could change that ofcourse.